test1/task1.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+char* removeSpaces(const char* string)
+{
+    size_t sizeOfString = strlen(string);
```

Comment:
По-хорошему 
```suggestion
    const size_t sizeOfString = strlen(string);
```
Вообще, const в C мало на что влияют, но могут, например, выступать в роли подсказки при оптимизациях, так что лучше их расставлять.